.github/workflows/merged.yml

```diff
@@ -150,6 +150,10 @@ jobs:
           GIT_TOKEN: ${{secrets.MODULE_FETCH_TOKEN}}
         run: |
           ./scripts-configs/store-access-token.sh
+          
+      - name: Export AppCheck Secret
+        run: |
+          echo APP_CHECK_DEBUG_TOKEN=${{ secrets.BUILD_APPCHECK_DEBUG_TOKEN }} >> ../secrets.xcconfig
```

Comment:
Looking at the docs, the example I sent you might be wrong, I think we need to declare it as an environment variable explicitly 🙂 
https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions#example-using-bash

Reply:
So are you saying I should replace this completely & we don't need to create the secrets file anymore? 😅

Reply:
We need to create the file, but we need to follow the pattern in the docs using `env` _and_ `run` together to get the secret:
```yaml
steps:
  - shell: bash
    env:
      SUPER_SECRET: ${{ secrets.SuperSecret }}
    run: |
      example-command "$SUPER_SECRET"
```